internal/bminventory/inventory.go

```diff
@@ -340,9 +341,35 @@ func (b *bareMetalInventory) setDefaultRegisterClusterParams(ctx context.Context
 		}
 	}
 
+	params.NewClusterParams.NetworkType, err = getDefaultNetworkType(params)
+	if err != nil {
+		return params, err
+	}
+
 	return params, nil
 }
 
+// If the cluster is SNO or the OpenShift version >= 4.12.0-0.0,
```

Comment:
Please adhere to [godoc](https://go.dev/blog/godoc) documentation format, specifically `Notice this comment is a complete sentence that begins with the name of the element it describes`